sound/smooth_swing_v2.h

```diff
@@ -144,11 +162,14 @@ class SmoothSwingV2 : public SaberBasePassThrough {
             STDOUT.print("  hum_volume: ");
             STDOUT.println(hum_volume);
           }
-	  if (on_) {
-	    // We need to stop setting the volume when off, or playback may never stop.
-	    A.set_volume(mixhum * mixab);
-	    B.set_volume(mixhum * (1.0 - mixab));
-	  }
+          if (on_) {
+            // We need to stop setting the volume when off, or playback may never stop.
+            if (accent_swings_present) {
```

Comment:
we don't need to check if accent swings are present here.
Calling SetSwingVolume should be safe even if accent swings are not present.

Reply:
It's safe provided that the defaults are set so that swing pairs are not ducked.  I'll make the required changes.